src/test/java/com/exasol/adapter/document/files/IntegrationTestSetup.java

```diff
@@ -164,13 +171,19 @@ private Map<String, String> getVirtualSchemaProperties(final String mapping) {
         if (!debugProperty.isBlank() || !profileProperty.isBlank()) {
             properties.put("MAX_PARALLEL_UDFS", "1");
         }
-        if (System.getProperty("test.vs-logs", "false").equals("true")) {
-            properties.put("DEBUG_ADDRESS", "127.0.0.1:3001");
-            properties.put("LOG_LEVEL", "ALL");
-        }
+        properties.putAll(debugProperties());
         return properties;
     }
 
+    private Map<String, String> debugProperties() {
+        final String debugAddress = System.getProperty("com.exasol.virtualschema.debug.address");
```

Comment:
In other VS we have the following properties:

* `com.exasol.log.host`
* `com.exasol.log.port`
* `com.exasol.log.level`

I think we should keep this uniform.